### PR TITLE
Onfido client asyncio support

### PR DESCRIPTION
**Pull request by jerive**
This MR brings asyncio support to onfido-triton (via aiohttp).
I had to drop support for python 3.6. Please let me know if this is an issue.

### CODE REVIEW

**Comment by jerive** [comment]
Fixed

**Review by RiccardoPetteruti-Onfido — APPROVED** [review]

File: onfido/resources_aio/applicants.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from ..aio_resource import Resource
+
+
+class Applicants(Resource):
+    def create(self, request_body: dict):
+        return self._post("applicants/", **request_body)
```
Comment (RiccardoPetteruti-Onfido) [inline onfido/resources_aio/applicants.py:6]: If the path "applicants/" is common this can be declared as a var.

File: onfido/onfido.py [hunk 2]
```diff
@@ -33,3 +45,23 @@ def __init__(self, api_token, region, timeout=None):
             pass
         elif "api.onfido.com" in region:
             raise OnfidoRegionError("The region must be one of Region.EU, Region.US or Region.CA. We previously defaulted to Region.EU, so if you previously didn’t set a region or used api.onfido.com, please set your region to Region.EU")
+
+class AsyncApi:
+    def __init__(self, api_token, region, aio_session, timeout=None):
+        self.applicant = AsyncApplicants(api_token, region, aio_session, timeout)
```
Comment (RiccardoPetteruti-Onfido) [inline onfido/onfido.py:51]: In aio_resource the private class vars are declared using _ as "private" vars, why here it's not done the same way?

Comment (jerive) [reply onfido/onfido.py:51]: same here, chamaleon
Or I don't understand what you mean ?

Comment (RiccardoPetteruti-Onfido) [reply onfido/onfido.py:51]: nevermind, i missread here

**Review by RiccardoPetteruti-Onfido — APPROVED** [review]

**Review by DavidMealha-Onfido — APPROVED** [review]

**Comment by dvacca-onfido** [comment]
> This MR brings asyncio support to onfido-triton (via aiohttp). I had to drop support for python 3.6. Please let me know if this is an issue.

I think it's fine dropping support for python 3.6 as it reached EOL

**Review by dvacca-onfido — APPROVED** [review]

**Pull request merged**